src/cls/user/cls_user.cc

```diff
@@ -419,6 +419,9 @@ static int cls_user_reset_stats(cls_method_context_t hctx,
 	return -EIO;
       }
       add_header_stats(&header.stats, e);
+      if (!keys.empty()) {
+        from_index = keys.rbegin()->first;
+      }
```

Comment:
shouldn't this go outside the `for (const auto& kv : keys) {}` loop?

Reply:
oops, C-c C-v error, fixing